src/backend/distributed/shared_library_init.c

```diff
@@ -1629,22 +1622,6 @@ RegisterCitusConfigVariables(void)
 		GUC_STANDARD,
 		NULL, NULL, NULL);
 
-	DefineCustomEnumVariable(
-		"citus.shard_placement_policy",
```

Comment:
quite interesting that we didn't have any files testing this guc (i.e.: set citus.shard_placement_policy to foo)